src/stacks-hierarchy/Root_PurchaseOverviewScreen/components/TravellerSelection.tsx

```diff
@@ -27,65 +33,145 @@ export function TravellerSelection({
 
   if (selectionMode === 'none') {
     return <></>;
   }
 
+  let activeTravellerCategoryCount = 0;
+  const totalTravellersCount = selectableUserProfilesWithCount.reduce(
+    (acc, sUPWC) => {
+      if (sUPWC.count > 0) {
+        activeTravellerCategoryCount += 1;
+      }
+      return acc + sUPWC.count;
+    },
+    0,
+  );
+
+  const multipleTravellerCategoriesSelectedFrom =
+    activeTravellerCategoryCount > 1;
+
+  const multipleTravellersDetailsText =
+    totalTravellersCount == 0
+      ? t(PurchaseOverviewTexts.travellerSelection.no_travellers_selected)
+      : selectableUserProfilesWithCount.reduce((acc, sUPWC) => {
+          if (sUPWC.count > 0) {
+            if (selectionMode == 'single') {
+              return getReferenceDataName(sUPWC, language);
+            }
+            if (acc !== '') {
+              acc += ', ';
+            }
+            return (
+              acc + sUPWC.count + ' ' + getReferenceDataName(sUPWC, language) // plural name values not ideal atm?
+            );
+          } else {
+            return acc;
+          }
+        }, '');
+
+  const travellerSelectionOnPress = () => {
+    openBottomSheet(() => (
+      <TravellerSelectionSheet
+        selectionMode={selectionMode}
+        fareProductType={fareProductType}
+        selectableUserProfilesWithCountInit={selectableUserProfilesWithCount}
+        close={(
+          chosenSelectableUserProfilesWithCounts?: UserProfileWithCount[],
+        ) => {
+          if (chosenSelectableUserProfilesWithCounts !== undefined) {
+            setSelectableUserProfilesWithCount(
+              chosenSelectableUserProfilesWithCounts,
+            );
+          }
+          closeBottomSheet();
+        }}
+      />
+    ));
+  };
+
   return (
     <View style={style}>
-      <InfoToggle
-        title={
-          selectionMode == 'multiple'
+      <View style={styles.sectionTitleContainer}>
+        <ThemeText type="body__secondary" color="secondary">
+          {selectionMode == 'multiple'
             ? t(PurchaseOverviewTexts.travellerSelection.title_multiple)
-            : t(PurchaseOverviewTexts.travellerSelection.title_single)
-        }
-        accessibilityLabel={t(
-          PurchaseOverviewTexts.infoToggle.travellerA11yLabel,
-        )}
-      />
-      {selectionMode === 'multiple' ? (
-        <MultipleTravellersSelection
-          fareProductType={fareProductType}
-          {...userCountState}
-          userProfilesWithCount={selectableUserProfilesWithCount}
-        />
-      ) : (
-        <SingleTravellerSelection
-          fareProductType={fareProductType}
-          {...userCountState}
-          userProfilesWithCount={selectableUserProfilesWithCount}
-        />
-      )}
+            : t(PurchaseOverviewTexts.travellerSelection.title_single)}
+        </ThemeText>
+      </View>
+      <Section {...accessibility}>
+        <GenericClickableSectionItem onPress={travellerSelectionOnPress}>
```

Comment:
Some overflow when large text. Usually fixed by putting `flex: 1` somewhere.

<img width=400 src='https://github.com/AtB-AS/mittatb-app/assets/675421/2c9585ac-04ee-4574-9b58-b1aae484cf19' />

Reply:
{flex: 1} did the trick